sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/ThriftServerQueryTestSuite.scala

```diff
@@ -103,7 +103,10 @@ class ThriftServerQueryTestSuite extends SQLQueryTestSuite with SharedThriftServ
     // SPARK-42921
     "timestampNTZ/datetime-special-ansi.sql",
     // SPARK-47264
-    "collations.sql"
+    "collations.sql",
+    "binary_hex.sql",
+    "binary_basic.sql",
+    "binary_base64.sql"
```

Comment:
to double check: we ignore these tests because they also use string collation? @yaooqinn

Reply:
This relies on HiveJDBC as we currently pass raw bytes to the client side

Reply:
Ah I see. It's better if we can add comments to explain it.

Reply:
SGTM, Let me add some

Reply:
It looks like we fetch them from the result set as Object and can be formatted on our own. These can be activated actually.